src/BenchmarksServer/Git.cs

```diff
@@ -8,6 +8,8 @@ namespace BenchmarkServer
 {
     public static class Git
     {
+        private const int CommandTimeout = 5000;
```

Comment:
Change to `TimeSpan`.  Also, cloning a large repo may take a fairly long time.  Would 30 seconds be a better timeout?